src/test/unit/DelegationUnit.t.sol

```diff
@@ -43,6 +43,8 @@ contract DelegationManagerUnitTests is EigenLayerUnitTestSetup, IDelegationManag
     address defaultStaker = cheats.addr(uint256(123_456_789));
     address defaultOperator = address(this);
 
+    uint256 initializedWithdrawalDelayBlocks = 0;
```

Comment:
should this test or any others set this to a nonzero value?
feels harder to test that the functionality is correct if we are setting this to zero, I think.

Reply:
I left it as 0 since we never changed it from its default 0 value in the unit tests. 
Updated now to a nonzero value but tests still seem to pass because we never added back the unit tests for the withdrawals. Will add them back to `DelegationUnit.t.sol` in a followup PR